src/components/EventCard/styles.less

```diff
@@ -13,80 +13,84 @@
   scrollbar-width: none;
   width: @event-width;
 
-  .edit-icon-wrapper {
-    font-size: 24px;
-    position: absolute;
-    right: 10px;
-    top: 10px;
-
-    .edit-icon {
-      color: @black;
-      cursor: pointer;
-      transition: all 0.2s;
-    }
+  .front-facing {
+    min-height: 100%;
 
-    .edit-icon:hover {
-      color: @gray-4;
+    .edit-icon-wrapper {
+      font-size: 24px;
+      position: absolute;
+      right: 10px;
+      top: 10px;
+
+      .edit-icon {
+        color: @black;
+        cursor: pointer;
+        transition: all 0.2s;
+      }
+
+      .edit-icon:hover {
+        color: @gray-4;
+      }
     }
-  }
 
-  img {
-    height: 150px;
-    margin: 0;
-    object-fit: cover;
-    width: 100%;
-  }
-
-  .info {
-    padding: 5% 10% 0 10%;
```

Comment:
If you cut the top padding here, there should be enough room to squeeze 2-3 lines of title + include the bar at the bottom.

Reply:
Removing the top padding makes the text be a bit too close to the event cover, but perhaps it's fine for a while.